python/lib/dependabot/python/helpers.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "time"
+require "open3"
+
+require "dependabot/errors"
+require "dependabot/shared_helpers"
+
+module Dependabot
+  module Python
+    module Helpers
+      def self.run_poetry_command(command, fingerprint: nil)
+        start = Time.now
+        command = SharedHelpers.escape_command(command)
+        stdout, process = Open3.capture2e(command)
+        time_taken = Time.now - start
+
+        # Raise an error with the output from the shell session if Pipenv
```

Comment:
Hmm... I think `Pipenv` was a typo here, and it should actually be `poetry`...

However, nothing in this logic (that I can see) is `poetry` specific, so maybe this should be `run_python_package_manager_command()`??

As we may want to use this for `pipenv`, `pip-tools`, `pip`, etc?

Reply:
```suggestion
        # Raise an error with the output from the shell session if Poetry
```